ksql-engine/src/main/java/io/confluent/ksql/function/udaf/topk/TopkAggFunctionDeterminer.java

```diff
@@ -24,30 +24,57 @@
 import io.confluent.ksql.function.KsqlAggFunctionDeterminer;
 import io.confluent.ksql.function.KsqlAggregateFunction;
 import io.confluent.ksql.util.KsqlException;
+import org.apache.kafka.connect.data.SchemaBuilder;
 
 public class TopkAggFunctionDeterminer extends KsqlAggFunctionDeterminer {
+  private int topKSize = 0;
 
   public TopkAggFunctionDeterminer() {
     super("TOPK", Arrays.asList());
```

Comment:
Would be better if this ctor delegates to the new one you have added. And then the new one calls `super`